gc/base/standard/Scavenger.cpp

```diff
@@ -5552,10 +5560,45 @@ MM_Scavenger::threadReleaseCaches(MM_EnvironmentBase *currentEnvBase, MM_Environ
 		flushInactiveDeferredCopyScanCache(currentEnv, targetEnv, flushCaches, final);
 		deactivateDeferredCopyScanCache(currentEnv, targetEnv, flushCaches, final);
 
+		if (flushCaches && (MUTATOR_THREAD == targetEnv->getThreadType()) && isCurrentPhaseConcurrent()) {
+			/* Flush copy byte stats from Mutator threads during the concurrent phase (via Read Barrier).
+			 * GC threads will merge the stats a bit later (STW or concurrent) when the rest of the stats are merged.
+			 * These flushes are minimizeed to only when flushCaches is true, which should be typically just
+			 * a couple of times at the end of the concurrent phase. Hence, invocations from OOL VM Access Release will be ingored.
```

Comment:
fixed typos: minimizeed → minimized, ingored → ignored
```suggestion
			/* Flush copy byte stats from Mutator threads during the concurrent phase (via Read Barrier).
			 * GC threads will merge the stats a bit later (STW or concurrent) when the rest of the stats are merged.
			 * These flushes are minimized to only when flushCaches is true, which should be typically just
			 * a couple of times at the end of the concurrent phase. Hence, invocations from OOL VM Access Release will be ignored.
```